internal/internal.go

```diff
@@ -189,11 +190,20 @@ func (p *envoyExtAuthzGrpcServer) listen() {
 
 func (p *envoyExtAuthzGrpcServer) Check(ctx ctx.Context, req *ext_authz.CheckRequest) (resp *ext_authz.CheckResponse, err error) {
 	start := time.Now()
+
+	result := evalResult{}
+	result.metrics = metrics.New()
+	result.metrics.Timer(metrics.ServerHandler).Start()
+	result.decisionID, err = uuid4()
+	if err != nil {
+		return nil, err
```

Comment:
Nit: I think case warrants a log message (at error level). If the server is unable to generate a UUID something is quite broken and it would be useful to see it in the logs. We can't send a decision log (because the ID can't be generated but at least someone looking at the console logs would be able to see the problem.)

Reply:
I guess it's possible that the gRPC server will log an error for us? I'm not sure how that works.

Reply:
Yes the gRPC server will report this error.

Reply:
It'll log it? Can we rely on that?

Reply:
I've seen it fail with the error reported. I'll add a log as well just in case.